tests/gtest/datahandle_attribute_test.cpp

```diff
@@ -91,43 +103,75 @@ class DatahandleAttributeTest : public ::testing::Test {
         auto xline_distance_x = cdp[3].first - cdp[0].first;
         auto xline_distance_y = cdp[3].second - cdp[0].second;
 
-        const double xori = cdp[0].first;
-        const double yori = cdp[0].second;
+        double xori = cdp[0].first;
+        double yori = cdp[0].second;
         const double xinc = std::hypot(iline_distance_x, iline_distance_y) / nsteps_iline;
         const double yinc = std::hypot(xline_distance_x, xline_distance_y) / nsteps_xline;
         const double rotation = std::atan2(iline_distance_y, iline_distance_x) * 180 / M_PI;
 
+        xori += iline_distance_x / nsteps_iline * origin_steps_shift_iline +
```

Comment:
Why change the order of the inline and line operands between the lines? Would be a lot easier to read this if it was the same.